docs/ref/booking/create.rst

```diff
@@ -1,7 +1,11 @@
 booking/create
---------------
+==============
 A call to booking/create must pass in at least one :ref:`slip` - either directly as an array of slips, or by using a **session_id** from a previously created :doc:`session` containing at least one :ref:`slip`. Additionally, the call must pass in any customer input entered to fields from :doc:`form`, for all required fields and any optional fields.
 
+Public API
+----------
+If the Public API is enabled, non-authenticated users can make requests to the */booking/create* end point. This is the same functionality as a customer making a booking through the UI on the `Hosted Booking Page <https://support.checkfront.com/hc/en-us/articles/115004917593-Hosted-Booking-Page>`_.
```

Comment:
endpoint should be one word, and I would maybe say `unauthenticated` rather than `non-authenticated`.

Reply:
I went through all the changed docs to make sure non-authenticated was replaced with unauthenticated. Resolved in the latest commit.